mods/ra/rules/infantry.yaml

```diff
@@ -238,13 +238,15 @@ SPY:
 		Name: Spy
 		GenericName: Soldier
 	-Guard:
+	Mobile:
+		Voice: Move
 	RevealsShroud:
 		Range: 5c0
 	Passenger:
 		PipType: Yellow
 		Voice: Move
 	Disguise:
-		Voice: Move
+		Voice: Action
```

Comment:
`Action` should be the default (see [here](https://github.com/OpenRA/OpenRA/blob/4dba9f5b888c286938e59245322a1b29708f5ef4/OpenRA.Mods.Cnc/Traits/Disguise.cs#L77)), so this can be removed.